### ISSUE TITLE
OSError: [Errno 22] on Detectron2LayoutModel()

### ISSUE THREAD

**Issue by FuJiK0** [Layout-Parser/layout-parser#140]
When I try to Create a Detectron2-based Layout Detection Model, I encounter the following error:
```python
model = lp.Detectron2LayoutModel(
    'lp://PrimaLayout/mask_rcnn_R_50_FPN_3x/config',
    extra_config=["MODEL.ROI_HEADS.SCORE_THRESH_TEST", 0.5],
    label_map={
        1: "TextRegion",
        2: "ImageRegion"
    })
```
Error:
```python
File f:\Coding\Conda\envs\LP\lib\site-packages\portalocker\utils.py:289, in Lock._get_fh(self)
    [287](file:///f%3A/Coding/Conda/envs/LP/lib/site-packages/portalocker/utils.py?line=286) def _get_fh(self) -> typing.IO:
    [288](file:///f%3A/Coding/Conda/envs/LP/lib/site-packages/portalocker/utils.py?line=287)     '''Get a new filehandle'''
--> [289](file:///f%3A/Coding/Conda/envs/LP/lib/site-packages/portalocker/utils.py?line=288)     return open(self.filename, self.mode, **self.file_open_kwargs)

OSError: [Errno 22] Invalid argument: 'C:\\Users\\60953/.torch/iopath_cache\\s/yc92x97k50abynt\\config.yaml?dl=1.lock'
```
![image](https://user-images.githubusercontent.com/30752880/169946534-c4f3089c-dce9-4999-827b-d49594aa9eac.png)

![image](https://user-images.githubusercontent.com/30752880/169946478-621ebd78-be93-4fc0-8999-40d6b5881e34.png)

**Environment**
1.  Platform ```Windows 11```
2. Layout Parser version ```0.3.4```
3. Torch version ```1.10.0+cu113```

**Comment by jiboncom** [comment]
I am also experiencing this in the same environment

**Comment by Micheduc25** [comment]
> I am also experiencing this in the same environment. It seems to be related to a problem downloading files from the LP API because when downloading the config files and pointing LP to use the local file the error is mentions `model_final.pth?dl=1odel_final.pth?dl=1` .

Have you been able to find a solution so far?

**Comment by jiboncom** [comment]
Not really. I did a few experiments I needed to run using Google Colab but I couldn't fix the problem in my computer. It seems to be related to the long path issue in Windows but allowing that from registry didn't fix the problem. I didn't try running python on "higher" level folder (I am using Vs Code) but read that it could help.

**Comment by lolipopshock** [comment]
This seems to be a known issue for Windows users and we plan to implement some important fixes in the next version.

**Comment by MarouaneZ1** [comment]
i'm trying to use detectron2 model for extraction information but i got this problem, i tried many ways but i failed to solve it,
i tried to use google collab but it throws me :"ImportError: you cannot import is_directory",
does anyone know how can i fix it ?
any help would be appreciated

<img width="724" alt="erreurDetectron" src="https://user-images.githubusercontent.com/98398821/192057764-0d62e6f6-5f13-4941-8ad1-adb19b51b752.PNG">
<img width="677" alt="erreurDetectronn" src="https://user-images.githubusercontent.com/98398821/192057768-fe5d4a39-aea0-41fd-b172-45594d09ccbf.PNG">

**Comment by MarouaneZ1** [comment]
> This seems to be a known issue for Windows users and we plan to implement some important fixes in the next version. Stay tuned -- thanks!

did you implement new important fixes yet ? bcz i'm hard stuck on an error : OSerror : invalid argument

**Comment by zellsophia** [comment]
any updates on this issue? Coz I'm still running into it

**Comment by uzairaslam19** [comment]
Any Updates on this? I'm running into same error

**Comment by 1042312930** [comment]
> Any Updates on this? I'm running into same error

i find this in
https://github.com/Layout-Parser/layout-parser/issues/15
![image](https://github.com/Layout-Parser/layout-parser/assets/70195780/525baaa7-39da-4b43-8c7d-d8bdef327995)
